src/Underworld.ts

```diff
@@ -2665,6 +2671,7 @@ ${CardUI.cardListToImages(player.stats.longestSpell)}
   async endFullTurnCycle() {
     // Increment the turn number now that it's starting over at the first phase
     this.turn_number++;
+    this.allyNPCAttemptWinKillSwitch++;
```

Comment:
Yes it used to increment the killswitch inside of the initializeTurnPhase function, but I'm not seeing it in there anymore (please double check - I'm currently on mobile and cannot ctrl+f for "killswitch")

I noticed some weird behavior with the kill switch during the refactor: it would increment every npc ally turn, even when the players could act. This basically means allies would give up on turn [killswitch], rather than fighting for [killswitch] turns after players have died, which felt inconsistent and buggy, especially if you died in later turns.

At some point during the refactor I was moving this logic around looking for a better way to write it and eventually I forgot or became distracted with another issue and never added it back in :P